### ISSUE TITLE
Cannot add node to multiple pools. 

### ISSUE THREAD

**Issue by brstallings** [F5Networks/f5-ansible#1595]
##### ISSUE TYPE

 - Bug Report

##### COMPONENT NAME

bigip_pool_member

##### ANSIBLE VERSION

```
ansible --version
ansible 2.8.5
  config file = /home/brstallings_solutions_corelogic_/clgx-fnc-ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.12 (default, Oct  8 2019, 14:14:10) [GCC 5.4.0 20160609]


```

##### PYTHON VERSION

```
Python 2.7.12
```

##### BIGIP VERSION

```
BIG-IP v14.1.0.1 (Build 0.0.7)
```

##### SUMMARY

Trying to add a web server node to 2 different F5 pools.  Since a web server can serve multiple websites at once via different ports, I'm trying to add a node to 2 or more different F5 pools. 
I started out telling ansible to create the node first using bigip_node module with the same outcome, but after reading multiple support incidents here on github, I started setting the node with the bigip_pool_member module instead.  It will create and add the node to the 1st pool in the script as expected.  As soon as it hits the second node in the script, it throws the following error: 
"item": "fncuc1gbbigip02-f61l",
    "msg": "0107003a:3: Pool member node (/Internal/sbxwebtest2_internal.app/fnc106test01) and existing node (/Internal/sbxwebtest_internal.app/fnc106test01) cannot use the same IP Address (10.112.124.106)."

I tried using reuse_nodes: yes and also no with the same outcome. I found this in the following support url: https://github.com/F5Networks/f5-ansible/issues/823

If I perform the following manually, it will work just fine:
1. On the F5, go to the 2nd pool and click Members tab.
2. Then click Add to add node.
3. Under the Address field, click the "Node List" radio button.
4. Select the fnc106test01 node from the drop down list.
5. Enter 2443 for the Service Port
6. Click Finish button.

I would assume if I could use the "Node List" parameter, then I could add the node to the F5 using the bigip_node module.  Then for each pool, I could use the "Node List" parameter to set the node and the service port.  

From within the bigip_pool_member module, how do I tell it to use "Node List" instead of "New Address" or "New FQDN Node"?  As I found out above, using "reuse_nodes" parameter doesn't work in this situation.

##### STEPS TO REPRODUCE

```
# Add Pool Members
- name: ADD POOL MEMBERS
  bigip_pool_member:
    provider: "{{hostvars[item].provider}}"
    partition: "{{hostvars[inventory_hostname].f5_partition}}/{{hostvars[inventory_hostname].f5_pool_name}}.app"
    address: "{{hostvars[inventory_hostname].networkInterfaces[0].networkIP}}"
    #host: "{{hostvars[inventory_hostname].networkInterfaces[0].networkIP}}"
    #address: "{{hostvars[inventory_hostname].inventory_hostname}}.c.clgx-fnc-glb-sbx-a94c.internal"
    name: "{{hostvars[inventory_hostname].inventory_hostname}}"
    port: "1443"
    pool: "{{hostvars[inventory_hostname].f5_pool_name}}_pool"
    state: present
    reuse_nodes: yes
  delegate_to: "{{item}}"
  delegate_facts: True
  loop: "{{groups._f5}}"

- name: ADD POOL MEMBERS
  bigip_pool_member:
    provider: "{{hostvars[item].provider}}"
    partition: "{{hostvars[inventory_hostname].f5_partition}}/{{hostvars[inventory_hostname].f5_pool_name2}}.app"
    address: "{{hostvars[inventory_hostname].networkInterfaces[0].networkIP}}"
    #host: "{{hostvars[inventory_hostname].networkInterfaces[0].networkIP}}"
    name: "{{hostvars[inventory_hostname].inventory_hostname}}"
    port: "2443"
    pool: "{{hostvars[inventory_hostname].f5_pool_name2}}_pool"
    state: present
    reuse_nodes: yes
  delegate_to: "{{item}}"
  delegate_facts: True
  loop: "{{groups._f5}}"
```

##### EXPECTED RESULTS

That it would add the node using port 1443 to the first pool and then add the node using port 2443 to the second pool. 
##### ACTUAL RESULTS

It threw the following error: 
"item": "fncuc1gbbigip02-f61l",
    "msg": "0107003a:3: Pool member node (/Internal/sbxwebtest2_internal.app/fnc106test01) and existing node (/Internal/sbxwebtest_internal.app/fnc106test01) cannot use the same IP Address (10.112.124.106)."

```
root@fnccsappansible-jumpbox:/home/brstallings_solutions_corelogic_/clgx-fnc-ansible# time ansible-playbook --extra-vars 'hosts=fnc106test01' sbx/pb_sbx_web_test_install.yml -vvvv
ansible-playbook 2.8.5
  config file = /home/brstallings_solutions_corelogic_/clgx-fnc-ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible
  executable location = /usr/local/bin/ansible-playbook
  python version = 2.7.12 (default, Oct  8 2019, 14:14:10) [GCC 5.4.0 20160609]
Using /home/brstallings_solutions_corelogic_/clgx-fnc-ansible/ansible.cfg as config file
setting up inventory plugins
host_list declined parsing /home/brstallings_solutions_corelogic_/clgx-fnc-ansible/inventory.gcp.yml as it did not pass it's verify_file() method
script declined parsing /home/brstallings_solutions_corelogic_/clgx-fnc-ansible/inventory.gcp.yml as it did not pass it's verify_file() method
Parsed /home/brstallings_solutions_corelogic_/clgx-fnc-ansible/inventory.gcp.yml inventory source with auto plugin
Loading callback plugin default of type stdout, v2.0 from /usr/local/lib/python2.7/dist-packages/ansible/plugins/callback/default.pyc

PLAYBOOK: pb_sbx_web_test_install.yml ****************************************************************************************************************************************
Positional arguments: sbx/pb_sbx_web_test_install.yml
become_method: sudo
inventory: (u'/home/brstallings_solutions_corelogic_/clgx-fnc-ansible/inventory.gcp.yml',)
forks: 5
tags: (u'all',)
extra_vars: (u'hosts=fnc106test01',)
verbosity: 4
connection: smart
timeout: 10
1 plays in sbx/pb_sbx_web_test_install.yml
 [WARNING]: Found variable using reserved name: hosts


PLAY [sbx_web_test playbook configure web server] ****************************************************************************************************************************
META: ran handlers

TASK [web/test/test_install : Collect facts] *********************************************************************************************************************************
task path: /home/brstallings_solutions_corelogic_/clgx-fnc-ansible/sbx/roles/web/test/test_install/tasks/main.yml:15
Using module file /usr/local/lib/python2.7/dist-packages/ansible/modules/windows/setup.ps1
Pipelining is enabled.
<fnc106test01> ESTABLISH WINRM CONNECTION FOR USER: Administrator on PORT 5986 TO fnc106test01
EXEC (via pipeline wrapper)
ok: [fnc106test01]

TASK [f5/f5_add_node : ADD POOL MEMBERS] *************************************************************************************************************************************
task path: /home/brstallings_solutions_corelogic_/clgx-fnc-ansible/sbx/roles/f5/f5_add_node/tasks/main.yml:52
<fncuc1gbbigip01-q19v> connection transport is rest
<fncuc1gbbigip01-q19v> ESTABLISH LOCAL CONNECTION FOR USER: root
<fncuc1gbbigip01-q19v> EXEC /bin/sh -c 'echo ~root && sleep 0'
<fncuc1gbbigip01-q19v> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /root/.ansible/tmp/ansible-tmp-1576116172.78-179463878272231 `" && echo ansible-tmp-1576116172.78-179463878272231="` echo /root/.ansible/tmp/ansible-tmp-1576116172.78-179463878272231 `" ) && sleep 0'
<fnc106test01> Attempting python interpreter discovery
<fncuc1gbbigip01-q19v> EXEC /bin/sh -c 'echo PLATFORM; uname; echo FOUND; command -v '"'"'/usr/bin/python'"'"'; command -v '"'"'python3.7'"'"'; command -v '"'"'python3.6'"'"'; command -v '"'"'python3.5'"'"'; command -v '"'"'python2.7'"'"'; command -v '"'"'python2.6'"'"'; command -v '"'"'/usr/libexec/platform-python'"'"'; command -v '"'"'/usr/bin/python3'"'"'; command -v '"'"'python'"'"'; echo ENDFOUND && sleep 0'
<fncuc1gbbigip01-q19v> EXEC /bin/sh -c '/usr/bin/python && sleep 0'
Using module file /usr/local/lib/python2.7/dist-packages/ansible/modules/network/f5/bigip_pool_member.py
<fncuc1gbbigip01-q19v> PUT /root/.ansible/tmp/ansible-local-8806XKRdWL/tmpMcVT7t TO /root/.ansible/tmp/ansible-tmp-1576116172.78-179463878272231/AnsiballZ_bigip_pool_member.py
<fncuc1gbbigip01-q19v> EXEC /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-tmp-1576116172.78-179463878272231/ /root/.ansible/tmp/ansible-tmp-1576116172.78-179463878272231/AnsiballZ_bigip_pool_member.py && sleep 0'
<fncuc1gbbigip01-q19v> EXEC /bin/sh -c '/usr/bin/python /root/.ansible/tmp/ansible-tmp-1576116172.78-179463878272231/AnsiballZ_bigip_pool_member.py && sleep 0'
<fncuc1gbbigip01-q19v> EXEC /bin/sh -c 'rm -f -r /root/.ansible/tmp/ansible-tmp-1576116172.78-179463878272231/ > /dev/null 2>&1 && sleep 0'
ok: [fnc106test01 -> fncuc1gbbigip01-q19v] => (item=fncuc1gbbigip01-q19v) => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "ansible_loop_var": "item",
    "changed": false,
    "deprecations": [
        {
            "msg": "Distribution Ubuntu 16.04 on host fnc106test01 should use /usr/bin/python3, but is using /usr/bin/python for backward compatibility with prior Ansible releases. A future Ansible release will default to using the discovered platform python for this host. See https://docs.ansible.com/ansible/2.8/reference_appendices/interpreter_discovery.html for more information",
            "version": "2.12"
        }
    ],
    "invocation": {
        "module_args": {
            "address": "10.112.124.106",
            "aggregate": null,
            "auth_provider": null,
            "availability_requirements": null,
            "connection_limit": null,
            "description": null,
            "fqdn": null,
            "fqdn_auto_populate": null,
            "ip_encapsulation": null,
            "monitors": null,
            "name": "fnc106test01",
            "partition": "Internal/sbxwebtest_internal.app",
            "password": null,
            "pool": "sbxwebtest_internal_pool",
            "port": 1443,
            "preserve_node": null,
            "priority_group": null,
            "provider": {
                "auth_provider": null,
                "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
                "proxy_to": null,
                "server": "fncuc1gbbigip01-q19v",
                "server_port": 8443,
                "ssh_keyfile": null,
                "timeout": null,
                "transport": "rest",
                "user": "admin",
                "validate_certs": false
            },
            "rate_limit": null,
            "ratio": null,
            "replace_all_with": false,
            "reuse_nodes": true,
            "server": null,
            "server_port": null,
            "state": "present",
            "transport": null,
            "user": null,
            "validate_certs": null
        }
    },
    "item": "fncuc1gbbigip01-q19v",
    "monitors": [],
    "state": "disabled"
}
<fncuc1gbbigip02-f61l> connection transport is rest
<fncuc1gbbigip02-f61l> ESTABLISH LOCAL CONNECTION FOR USER: root
<fncuc1gbbigip02-f61l> EXEC /bin/sh -c 'echo ~root && sleep 0'
<fncuc1gbbigip02-f61l> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /root/.ansible/tmp/ansible-tmp-1576116174.12-147453869738921 `" && echo ansible-tmp-1576116174.12-147453869738921="` echo /root/.ansible/tmp/ansible-tmp-1576116174.12-147453869738921 `" ) && sleep 0'
Using module file /usr/local/lib/python2.7/dist-packages/ansible/modules/network/f5/bigip_pool_member.py
<fncuc1gbbigip02-f61l> PUT /root/.ansible/tmp/ansible-local-8806XKRdWL/tmpqGhx_o TO /root/.ansible/tmp/ansible-tmp-1576116174.12-147453869738921/AnsiballZ_bigip_pool_member.py
<fncuc1gbbigip02-f61l> EXEC /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-tmp-1576116174.12-147453869738921/ /root/.ansible/tmp/ansible-tmp-1576116174.12-147453869738921/AnsiballZ_bigip_pool_member.py && sleep 0'
<fncuc1gbbigip02-f61l> EXEC /bin/sh -c '/usr/bin/python /root/.ansible/tmp/ansible-tmp-1576116174.12-147453869738921/AnsiballZ_bigip_pool_member.py && sleep 0'
<fncuc1gbbigip02-f61l> EXEC /bin/sh -c 'rm -f -r /root/.ansible/tmp/ansible-tmp-1576116174.12-147453869738921/ > /dev/null 2>&1 && sleep 0'
ok: [fnc106test01 -> fncuc1gbbigip02-f61l] => (item=fncuc1gbbigip02-f61l) => {
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "address": "10.112.124.106",
            "aggregate": null,
            "auth_provider": null,
            "availability_requirements": null,
            "connection_limit": null,
            "description": null,
            "fqdn": null,
            "fqdn_auto_populate": null,
            "ip_encapsulation": null,
            "monitors": null,
            "name": "fnc106test01",
            "partition": "Internal/sbxwebtest_internal.app",
            "password": null,
            "pool": "sbxwebtest_internal_pool",
            "port": 1443,
            "preserve_node": null,
            "priority_group": null,
            "provider": {
                "auth_provider": null,
                "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
                "proxy_to": null,
                "server": "fncuc1gbbigip02-f61l",
                "server_port": 8443,
                "ssh_keyfile": null,
                "timeout": null,
                "transport": "rest",
                "user": "admin",
                "validate_certs": false
            },
            "rate_limit": null,
            "ratio": null,
            "replace_all_with": false,
            "reuse_nodes": true,
            "server": null,
            "server_port": null,
            "state": "present",
            "transport": null,
            "user": null,
            "validate_certs": null
        }
    },
    "item": "fncuc1gbbigip02-f61l",
    "monitors": [],
    "state": "disabled"
}
[DEPRECATION WARNING]: Distribution Ubuntu 16.04 on host fnc106test01 should use /usr/bin/python3, but is using /usr/bin/python for backward compatibility with prior Ansible
 releases. A future Ansible release will default to using the discovered platform python for this host. See
https://docs.ansible.com/ansible/2.8/reference_appendices/interpreter_discovery.html for more information. This feature will be removed in version 2.12. Deprecation warnings
 can be disabled by setting deprecation_warnings=False in ansible.cfg.

TASK [f5/f5_add_node : ADD POOL MEMBERS] *************************************************************************************************************************************
task path: /home/brstallings_solutions_corelogic_/clgx-fnc-ansible/sbx/roles/f5/f5_add_node/tasks/main.yml:68
<fncuc1gbbigip01-q19v> connection transport is rest
<fncuc1gbbigip01-q19v> ESTABLISH LOCAL CONNECTION FOR USER: root
<fncuc1gbbigip01-q19v> EXEC /bin/sh -c 'echo ~root && sleep 0'
<fncuc1gbbigip01-q19v> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /root/.ansible/tmp/ansible-tmp-1576116175.14-197879237459201 `" && echo ansible-tmp-1576116175.14-197879237459201="` echo /root/.ansible/tmp/ansible-tmp-1576116175.14-197879237459201 `" ) && sleep 0'
<fnc106test01> Attempting python interpreter discovery
<fncuc1gbbigip01-q19v> EXEC /bin/sh -c 'echo PLATFORM; uname; echo FOUND; command -v '"'"'/usr/bin/python'"'"'; command -v '"'"'python3.7'"'"'; command -v '"'"'python3.6'"'"'; command -v '"'"'python3.5'"'"'; command -v '"'"'python2.7'"'"'; command -v '"'"'python2.6'"'"'; command -v '"'"'/usr/libexec/platform-python'"'"'; command -v '"'"'/usr/bin/python3'"'"'; command -v '"'"'python'"'"'; echo ENDFOUND && sleep 0'
<fncuc1gbbigip01-q19v> EXEC /bin/sh -c '/usr/bin/python && sleep 0'
Using module file /usr/local/lib/python2.7/dist-packages/ansible/modules/network/f5/bigip_pool_member.py
<fncuc1gbbigip01-q19v> PUT /root/.ansible/tmp/ansible-local-8806XKRdWL/tmpQ3hPle TO /root/.ansible/tmp/ansible-tmp-1576116175.14-197879237459201/AnsiballZ_bigip_pool_member.py
<fncuc1gbbigip01-q19v> EXEC /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-tmp-1576116175.14-197879237459201/ /root/.ansible/tmp/ansible-tmp-1576116175.14-197879237459201/AnsiballZ_bigip_pool_member.py && sleep 0'
<fncuc1gbbigip01-q19v> EXEC /bin/sh -c '/usr/bin/python /root/.ansible/tmp/ansible-tmp-1576116175.14-197879237459201/AnsiballZ_bigip_pool_member.py && sleep 0'
<fncuc1gbbigip01-q19v> EXEC /bin/sh -c 'rm -f -r /root/.ansible/tmp/ansible-tmp-1576116175.14-197879237459201/ > /dev/null 2>&1 && sleep 0'
The full traceback is:
WARNING: The below traceback may *not* be related to the actual failure.
  File "/tmp/ansible_bigip_pool_member_payload_Q8lu4K/__main__.py", line 1643, in main
    results = mm.exec_module()
  File "/tmp/ansible_bigip_pool_member_payload_Q8lu4K/__main__.py", line 1078, in exec_module
    output = self.execute(self.module.params)
  File "/tmp/ansible_bigip_pool_member_payload_Q8lu4K/__main__.py", line 1158, in execute
    changed = self.present()
  File "/tmp/ansible_bigip_pool_member_payload_Q8lu4K/__main__.py", line 1173, in present
    return self.create()
  File "/tmp/ansible_bigip_pool_member_payload_Q8lu4K/__main__.py", line 1241, in create
    self.create_on_device()
  File "/tmp/ansible_bigip_pool_member_payload_Q8lu4K/__main__.py", line 1395, in create_on_device
    raise F5ModuleError(response['message'])

failed: [fnc106test01 -> fncuc1gbbigip01-q19v] (item=fncuc1gbbigip01-q19v) => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "address": "10.112.124.106",
            "aggregate": null,
            "auth_provider": null,
            "availability_requirements": null,
            "connection_limit": null,
            "description": null,
            "fqdn": null,
            "fqdn_auto_populate": null,
            "ip_encapsulation": null,
            "monitors": null,
            "name": "fnc106test01",
            "partition": "Internal/sbxwebtest2_internal.app",
            "password": null,
            "pool": "sbxwebtest2_internal_pool",
            "port": 2443,
            "preserve_node": null,
            "priority_group": null,
            "provider": {
                "auth_provider": null,
                "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
                "proxy_to": null,
                "server": "fncuc1gbbigip01-q19v",
                "server_port": 8443,
                "ssh_keyfile": null,
                "timeout": null,
                "transport": "rest",
                "user": "admin",
                "validate_certs": false
            },
            "rate_limit": null,
            "ratio": null,
            "replace_all_with": false,
            "reuse_nodes": true,
            "server": null,
            "server_port": null,
            "state": "present",
            "transport": null,
            "user": null,
            "validate_certs": null
        }
    },
    "item": "fncuc1gbbigip01-q19v",
    "msg": "0107003a:3: Pool member node (/Internal/sbxwebtest2_internal.app/fnc106test01) and existing node (/Internal/sbxwebtest_internal.app/fnc106test01) cannot use the same IP Address (10.112.124.106)."
}
<fncuc1gbbigip02-f61l> connection transport is rest
<fncuc1gbbigip02-f61l> ESTABLISH LOCAL CONNECTION FOR USER: root
<fncuc1gbbigip02-f61l> EXEC /bin/sh -c 'echo ~root && sleep 0'
<fncuc1gbbigip02-f61l> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /root/.ansible/tmp/ansible-tmp-1576116176.04-259018347478193 `" && echo ansible-tmp-1576116176.04-259018347478193="` echo /root/.ansible/tmp/ansible-tmp-1576116176.04-259018347478193 `" ) && sleep 0'
Using module file /usr/local/lib/python2.7/dist-packages/ansible/modules/network/f5/bigip_pool_member.py
<fncuc1gbbigip02-f61l> PUT /root/.ansible/tmp/ansible-local-8806XKRdWL/tmpjlv52Z TO /root/.ansible/tmp/ansible-tmp-1576116176.04-259018347478193/AnsiballZ_bigip_pool_member.py
<fncuc1gbbigip02-f61l> EXEC /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-tmp-1576116176.04-259018347478193/ /root/.ansible/tmp/ansible-tmp-1576116176.04-259018347478193/AnsiballZ_bigip_pool_member.py && sleep 0'
<fncuc1gbbigip02-f61l> EXEC /bin/sh -c '/usr/bin/python /root/.ansible/tmp/ansible-tmp-1576116176.04-259018347478193/AnsiballZ_bigip_pool_member.py && sleep 0'
<fncuc1gbbigip02-f61l> EXEC /bin/sh -c 'rm -f -r /root/.ansible/tmp/ansible-tmp-1576116176.04-259018347478193/ > /dev/null 2>&1 && sleep 0'
The full traceback is:
WARNING: The below traceback may *not* be related to the actual failure.
  File "/tmp/ansible_bigip_pool_member_payload_iWeIKT/__main__.py", line 1643, in main
    results = mm.exec_module()
  File "/tmp/ansible_bigip_pool_member_payload_iWeIKT/__main__.py", line 1078, in exec_module
    output = self.execute(self.module.params)
  File "/tmp/ansible_bigip_pool_member_payload_iWeIKT/__main__.py", line 1158, in execute
    changed = self.present()
  File "/tmp/ansible_bigip_pool_member_payload_iWeIKT/__main__.py", line 1173, in present
    return self.create()
  File "/tmp/ansible_bigip_pool_member_payload_iWeIKT/__main__.py", line 1241, in create
    self.create_on_device()
  File "/tmp/ansible_bigip_pool_member_payload_iWeIKT/__main__.py", line 1395, in create_on_device
    raise F5ModuleError(response['message'])

failed: [fnc106test01 -> fncuc1gbbigip02-f61l] (item=fncuc1gbbigip02-f61l) => {
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "address": "10.112.124.106",
            "aggregate": null,
            "auth_provider": null,
            "availability_requirements": null,
            "connection_limit": null,
            "description": null,
            "fqdn": null,
            "fqdn_auto_populate": null,
            "ip_encapsulation": null,
            "monitors": null,
            "name": "fnc106test01",
            "partition": "Internal/sbxwebtest2_internal.app",
            "password": null,
            "pool": "sbxwebtest2_internal_pool",
            "port": 2443,
            "preserve_node": null,
            "priority_group": null,
            "provider": {
                "auth_provider": null,
                "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
                "proxy_to": null,
                "server": "fncuc1gbbigip02-f61l",
                "server_port": 8443,
                "ssh_keyfile": null,
                "timeout": null,
                "transport": "rest",
                "user": "admin",
                "validate_certs": false
            },
            "rate_limit": null,
            "ratio": null,
            "replace_all_with": false,
            "reuse_nodes": true,
            "server": null,
            "server_port": null,
            "state": "present",
            "transport": null,
            "user": null,
            "validate_certs": null
        }
    },
    "item": "fncuc1gbbigip02-f61l",
    "msg": "0107003a:3: Pool member node (/Internal/sbxwebtest2_internal.app/fnc106test01) and existing node (/Internal/sbxwebtest_internal.app/fnc106test01) cannot use the same IP Address (10.112.124.106)."
}

PLAY RECAP *******************************************************************************************************************************************************************
fnc106test01               : ok=2    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0


real    0m12.274s
user    0m3.892s
sys     0m1.094s


```

**Comment by brstallings** [comment]
I have narrowed the problem down to only when I'm adding nodes to pools that are part of an iApp.
I tested this morning manually creating 2 pools and running my script to add the nodes to each pool.  That worked just fine.  
The problem is if the pool is part of an iApp, then it throws the "Pool member node and existing node cannot use the same IP Address."
If I manually go into both pools tied to the iApp and manually enter each node using the "Node List" radio button, I can successfully add the node to both pools.

**Comment by focrensh** [comment]
Do the iapps enforce strict updates?

For the module to reuse a node, the node name must match in the module to what it is currently on the box. If the iapp has named the node something different from the hostname you are using in the vars, this would be the expected error.

The below combination must match what the iapp is trying to use as well:

```
        address: "{{ item.ip }}"
        name: "{{ item.hn }}"
```

**Comment by brstallings** [comment]
So I found that if I create the pool first and then set the iApp to use that particular pool, then I can add nodes to multiple pools with no issues at all.  The issue seems to be that when I allow the iApp to create the pool, the Partition used is not just the partition name that I've set.  If the partition set is Internal and the iApps creates the pool, then the partition that the pool is built on is named Internal/"iapp_name".app.  I tried setting the bigip_member_pool_member partition to Internal/"iapp_name".app, but that doesn't work.
I can use the create pool first workaround, but wonder if it's a bug, limitiation, or parameter missing in the bigip_pool_member module that won't allow you to use "Node List" when adding a node.  That's the correct way to add a node to an iApp pool.
